tests/unit/test_utils.py

```diff
@@ -2218,7 +2218,7 @@ def get_s3_accesspoint_request(
         self,
         accesspoint_name=None,
         accesspoint_context=None,
-        **s3_request_kwargs
+        **s3_request_kwargs,
```

Comment:
Did a formatter prompt this change? It's kind of odd to have a comma here since this is a catchall for everything.

Reply:
I see this came from the automated linter checks, I'm curious why this wasn't caught before.